backend/signaling/src/payments/sold_codes.js

```
@@ -11,11 +11,103 @@
+  throw new Error("sold_code_store_lock_lost");
+}
+
+function releaseStoreLock(lock) {
+  try {
+    fs.closeSync(lock.descriptor);
+  } finally {
+    try {
+      const owner = fs.readFileSync(SOLD_LOCK_FILE, "utf8").trim();
+      if (owner.endsWith(`:${lock.ownerToken}`)) fs.unlinkSync(SOLD_LOCK_FILE);
+    } catch (error) {
+      if (error.code !== "ENOENT") throw error;
```

Biome (2.5.3)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)